src/qt/rpcconsole.cpp

```diff
@@ -1343,7 +1343,7 @@ void RPCConsole::banSelectedNode(int bantime)
 	// Find possible nodes, ban it and clear the selected node
 	const CNodeCombinedStats *stats = clientModel->getPeerTableModel()->getNodeStats(detailNodeRow);
 	if(stats) {
-            m_node.ban(stats->nodeStats.addr, BanReasonManuallyAdded, bantime);
+        m_node.ban(stats->nodeStats.addr, BanReasonManuallyAdded, bantime);
```

Comment:
we should keep upstream's extra spaces, but even if we did decide to drop them, this only has two spaces not four